diff -up openssl-1.1.1m/include/openssl/opensslv.h.10~ openssl-1.1.1m/include/openssl/opensslv.h
--- openssl-1.1.1m/include/openssl/opensslv.h.10~	2022-01-10 13:34:39.116865519 +0100
+++ openssl-1.1.1m/include/openssl/opensslv.h	2022-01-10 13:35:03.473950305 +0100
@@ -40,7 +40,7 @@ extern "C" {
  *  major minor fix final patch/beta)
  */
 # define OPENSSL_VERSION_NUMBER  0x101010dfL
-# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1m  14 Dec 2021"
+# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1m  FIPS 14 Dec 2021"
 
 /*-
  * The macros below are to be used for shared library (.so, .dll, ...)
