diff -up openssl-1.1.1h/include/openssl/opensslv.h.11~ openssl-1.1.1h/include/openssl/opensslv.h
--- openssl-1.1.1h/include/openssl/opensslv.h.11~	2020-10-03 02:02:52.331377150 +0200
+++ openssl-1.1.1h/include/openssl/opensslv.h	2020-10-03 02:03:18.438133115 +0200
@@ -40,7 +40,7 @@ extern "C" {
  *  major minor fix final patch/beta)
  */
 # define OPENSSL_VERSION_NUMBER  0x1010108fL
-# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1h  22 Sep 2020"
+# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1h FIPS  22 Sep 2020"
 
 /*-
  * The macros below are to be used for shared library (.so, .dll, ...)
